example/apriltag_demo.c

```diff
@@ -54,6 +60,10 @@ either expressed or implied, of the Regents of The University of Michigan.
 //
 // tagtest [options] input.pnm
 
+#ifdef _WIN32
+#define  hamm_hist_max 10
+#endif
+
```

Comment:
Why is this not a variable like for the Linux case? Use the same code unless it is required by the API.

Reply:
On older compilers, you couldn't set array sizes using const variable values, it needed to be a define. I've removed it since I don't think this has been an issue for many years.

Reply:
Nevermind, this needs to be a define to work with the Windows compiler. I've removed the const int declation, the #ifdefs and switched it to a #define.